packages/SketchMorph2-Core.package/M2ViewPort.class/instance/insertSelection.st

```diff
@@ -0,0 +1,8 @@
+tool selection
+insertSelection
+ 	super insertSelection.
+	self	
+		currentCursor: ((self clipboardForm magnifyBy: self scalar) 
+			offset: (self brushSize asPoint / 2 + (self clipboardForm extent * self scalar * 0.5) negated));
```

Comment:
Does `negated` apply to the whole calculation or just the last part after the `+`?